preprocessing.py

```diff
@@ -232,6 +232,27 @@ def get_tpu_input_tensors(batch_size, tf_records, num_repeats=1,
     return dataset
 
 
+def get_tpu_bt_input_tensors(batch_size, num_repeats=1,
+                             shuffle_records=True, shuffle_examples=True,
+                             shuffle_buffer_size=1024,
```

Comment:
for now lets drop all the parameters except batch_size and random_rotation

Reply:
Can do. Note that I'm also ignoring `random_rotation` in the present implementation because of the problem we saw (which I would still like to get fixed before submitting).